MediaRecorder.bs

```diff
@@ -680,68 +680,27 @@ maximum {{Blob}} size that it will support, or the number of
 {{MediaStreamTrack}}s it will record at once.
 It will signal a fatal error if these limits are exceeded.
 
-## MediaRecorderErrorEvent ## {#errorevent-section}
-
-The {{MediaRecorderErrorEvent}} interface is defined for cases when an event is
-raised that was caused by an error.
+## Error eventsMediaRecorderErrorEvent ## {#errorevent-section}
 
 To <dfn>fire an error event</dfn>
```

Comment:
Shouldn't this whole section be replaced with a reference to https://html.spec.whatwg.org/multipage/webappapis.html#the-event-handler-processing-algorithm ?

Reply:
I've simplified to make "fire an error event" mean "fire an event with ErrorEvent as constructor" which I think achieves what we need